server/src/main/java/org/elasticsearch/action/support/single/instance/TransportInstanceSingleOperationAction.java

```diff
@@ -49,8 +49,8 @@
 
 import java.util.function.Supplier;
 
-public abstract class TransportInstanceSingleOperationAction<Request extends InstanceShardOperationRequest<Request>, Response extends ActionResponse>
-        extends HandledTransportAction<Request, Response> {
+public abstract class TransportInstanceSingleOperationAction<Request extends InstanceShardOperationRequest<Request>,
+        Response extends ActionResponse> extends HandledTransportAction<Request, Response> {
```

Comment:
I'd prefer:

```suggestion
        Response extends ActionResponse> 
        extends HandledTransportAction<Request, Response> {
```

But again, either way really.

Reply:
```
public abstract class TransportInstanceSingleOperationAction<
            Request extends InstanceShardOperationRequest<Request>,
            Response extends ActionResponse
        > extends HandledTransportAction<Request, Response> {
```

Is my preference. Stately.